zcode/zcode_test.go

```diff
@@ -25,13 +25,12 @@ func TestAppendContainer(t *testing.T) {
 	for _, c := range appendCases {
```

Comment:
Rename TestAppendContainer to TestAppend and remove TestAppendPrimitive since they are now identical.